components/flipper.py

```diff
@@ -2,18 +2,25 @@
 
 
 class Flipper:
+    """this flips a bucket with an arm that has a spinning motor"""
+    
 
     flipper_motor = wpilib.Talon
     spinning = False
+    #It starts not spinning
```

Comment:
Can you read this please?
https://www.python.org/dev/peps/pep-0008/#inline-comments